modules/docs/arrow-docs/docs/docs/arrow/data/nonemptylist/README.md

```diff
@@ -136,7 +136,7 @@ map(nelId, nelName, nelYear) { (id, name, year) ->
 - `NonEmptyList` is __used to model lists that guarantee at least one element__
 - We can easily construct values of `NonEmptyList` with `NonEmptyList.of`
 - `foldLeft`, `map`, `flatMap` and others are used to compute over the internal contents of a `NonEmptyList` value.
-- `binding { ... } Comprehensions` can be __used to imperatively compute__ over multiple `NonEmptyList` values in sequence.
+- `fx { ... }  and binding { ... } Comprehensions` can be __used to imperatively compute__ over multiple `NonEmptyList` values in sequence.
```

Comment:
It's deprecated everywhere because it's annotated in its definition. I don't believe we should remove the reference to `binding` from that doc because it's deprecated but not gone and we already know the issues we have with doc versioning which @dominv and Watt from 47 are gonna help us address for 0.9.0